pages/1.13/security/ent/tls-ssl/exhibitor/index.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: layout.pug
+navigationTitle:  Securing Exhibitor with mutual TLS
+title: Securing Exhibitor with mutual TLS
+menuWeight: 500
+excerpt: Securing DC/OS with a TLS enabled Exhibitor ensemble
+enterprise: true
+---
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+By default, the Exhibitor HTTP service is open to any client that can reach port 8181 on a master node. This page describes a method for protecting the Exhibitor service from unauthorized access. Once enabled, HTTP clients must access Exhibitor through Admin Router; thus applying the Admin Router access control policy to the Exhibitor service.
+
+<p class="message--note"><strong>NOTE: </strong>When accessing Exhibitor through Admin Router (https://master_host/exhibitor), authenticated users must have the <i>dcos:adminrouter:ops:exhibitor</i> privilege</p>
+
+# Securing Exhibitor
+
+The strategy for securing Exhibitor is mutual TLS authentication. In order to secure Exhibitor you must first create a unique root CA certificate. This CA certificate is used to sign various end entity certificates for the Admin Router and Exhibitor services. Creating a public key infrastructure that outputs PEM and Java KeyStore formatted artifacts is not a trivial task. To make this processes easier, a simple tool has been created for producing the necessary files.
+
+<p class="message--note"><strong>NOTE: </strong>This guide is only compatible with clusters which use <i>static</i> master discovery, <i>master_http_loadbalancer</i> is not currently supported. (https://docs.mesosphere.com/1.13/installing/production/advanced-configuration/configuration-reference/#master-discovery-required)</p>
+
+## Using the tool
```

Comment:
❓ Can you say where I should use this - e.g. "This tool can be run on any computer" or "This tool must be run on a DC/OS node".

I think "Using the tool" is a heading which requires one to have read the previous paragraph.
Instead, consider "Creating TLS Artifacts".